lib/arel/middleware/railtie.rb

```diff
@@ -1,4 +1,16 @@
 module Arel
+  if defined? Rails::Railtie
```

Comment:
Looking forward to some tests around this! Dummy app? Rails Rspec? 

You can add the Rails gem to the appraisal `arel_gems` group, so it's testing specs with and without Rails being loaded.